website/docs/releases/2024/v2024.2.md

```diff
@@ -0,0 +1,10180 @@
+---
+title: Release 2024.2 (unreleased)
+slug: /releases/2024.2
+---
+
+:::::note
+2024.2 has not been released yet! We're publishing these release notes as a preview of what's to come, and for our awesome beta testers trying out release candidates.
+:::::
+
+## Highlights
+
+-   **Remote Access Control** <span class="badge badge--primary">Enterprise</span> Access machines over RDP, SSH and VNC from authentik
+
+-   **Audit log** <span class="badge badge--primary">Enterprise</span> See what fields were changed when objects are updated
+
+-   **Session location and network binding** Increase security by preventing session theft
+
+-   **Wizard to simplify creating applications and providers**
+
+## Breaking changes
+
+### Manual action is required
+
+-   **Tenants have been renamed to brands**
+
+    Tenants, which used to allow the changing of branding configuration and default flows and a couple other settings depending on which domain authentik was accessed from, have been renamed to _brands_. This is due to this release including full tenancy (see [here](#new-features))
+
+    Existing _tenant_ objects will automatically be renamed to _brand_ objects. The API endpoints associated with _brands_ have also been renamed.
+
+    Blueprints using `authentik_tenants.tenant` will need to be changed to use `authentik_brands.brand`.
+
+    See documentation for _brands_ [here](../../core/brands.md)
+
```

Comment:
The next two paragraphs are a bit confusing.. are they still part of the bullet item about Tenants, or separate? I think they are related, but for readability I'd combine those two into one paragraph. And... mve the link to the docs down to very last line of this bullet item.

Reply:
Also it says see above, where it should say see below where we talk about the system settings page